charts/lau-case-backend/values.preview.template.yaml

```diff
@@ -1,15 +1,15 @@
 java:
   environment:
-    LAU_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase }}"
-    LAU_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername }}"
-    LAU_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword }}"
-    LAU_DB_ADMIN_USERNAME: "{{ .Values.postgresql.postgresqlUsername }}"
-    LAU_DB_ADMIN_PASSWORD: "{{ .Values.postgresql.postgresqlPassword }}"
+    LAU_DB_NAME: "{{ .Values.postgresql.auth.database }}"
+    LAU_DB_USERNAME: "{{ .Values.postgresql.auth.username }}"
```

Comment:
Hi Priyanka, can you change LAU_DB_USERNAME to use the "lauuser" like lau-idam-backend, thanks. Functionally this still works but the lauuser has lower DB priviledges which is safer.

Reply:
Thanks Dave, will do that